docs/source/python/csv.rst

```diff
@@ -16,7 +16,7 @@
 .. under the License.
 
 .. currentmodule:: pyarrow.csv
-.. _csv:
+.. _py-csv:
```

Comment:
Note @jorisvandenbossche : we (meaning I :-)) took the bad habit of not prefixing reference names with their language. This won't be a problem anymore if we split project docs, but I had to disambiguate here.